packages/medusa-file-spaces/src/services/digital-ocean.js

```diff
@@ -84,6 +84,78 @@ class DigitalOceanService extends AbstractFileService {
       })
     })
   }
+
+  // uploadStream(
+  //   fileData,
+  //   fileStream
+  // ) {
+  //   aws.config.setPromisesDependency(null)
+  //   aws.config.update({
+  //     accessKeyId: this.accessKeyId_,
+  //     secretAccessKey: this.secretAccessKey_,
+  //     region: this.region_,
+  //     endpoint: this.endpoint_,
+  //   }, true)
+
+  //   const parsedFilename = fileData.name
+  //   const fileKey = `${parsedFilename}-${Date.now()}.csv`
+  //   const s3 = new aws.S3()
+  //   var params = {
+  //     ACL: "bucket-owner-full-control",
+  //     Bucket: this.bucket_,
+  //     Body: fileStream,
+  //     Key: fileKey,
+  //   }
+
+  //   return new Promise((resolve, reject) => {
+  //     s3.upload(params, (err, data) => {
+  //       if (err) {
+  //         reject(err)
+  //         return
+  //       }
+
+  //       if (this.spacesUrl_) {
+  //         resolve({ url: `${this.spacesUrl_}/${data.Key}` })
+  //       }
+
+  //       resolve({ url: data.Location })
+  //     })
+  //   })
+  // }
+
+  async getUploadStream(fileData) {
+    aws.config.setPromisesDependency(null)
+    aws.config.update(
+      {
+        accessKeyId: this.accessKeyId_,
+        secretAccessKey: this.secretAccessKey_,
+        region: this.region_,
+        endpoint: this.endpoint_,
+      },
+      true
+    )
+
+    var pass = new stream.PassThrough()
+
+    const parsedFilename = fileData.name
+    const fileKey = `${parsedFilename}-${Date.now()}.csv`
+    const s3 = new aws.S3()
```

Comment:
suggestion: Always declare the variable as close as possible from where they are used :)